ext/sockets/sockets.c

```diff
@@ -535,6 +535,7 @@ static PHP_MINIT_FUNCTION(sockets)
 	REGISTER_LONG_CONSTANT("SO_LISTENQLIMIT",       SO_LISTENQLIMIT,        CONST_CS | CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("SO_LISTENQLEN",       SO_LISTENQLEN,        CONST_CS | CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("SO_USER_COOKIE",       SO_USER_COOKIE,        CONST_CS | CONST_PERSISTENT);
+	REGISTER_LONG_CONSTANT("SO_ACCEPTFILTER",       SO_ACCEPTFILTER,        CONST_CS | CONST_PERSISTENT);
```

Comment:
should be ifdef here ?

Reply:
oh it s safe SO_USER_COOKIE and SO_ACCEPTFILER coexist and SO_ACCEPTFILTER flag exist since freebsd 4.0 (2000).

Reply:
so ... *magic* ... that's cool, I thought something like this might be going on ...